packaging/macos/sunshine_pr.rb

```diff
@@ -0,0 +1,68 @@
+require "language/node"
+
+class @PROJECT_NAME@ < Formula
+  desc "@PROJECT_DESCRIPTION@"
+  homepage "@PROJECT_HOMEPAGE_URL@"
+  url "@GITHUB_CLONE_URL@", branch: "@GITHUB_BRANCH@"
```

Comment:
I don't really understand the purpose of a separate file if this is the only change? I'd also really like to avoid a separate file if possible. Ideally we validate the "production" file on PRs, with the only difference being what get's checked out.

Even though the other one is using "tag" it should still checkout the right commit, unless I'm missing something.

Reply:
I agree on not having another file, actually, this commit shouldn't be in this PR, I was testing and forgot to remove.

But regarding why, it's so I can download the file from the open PR and run it without changing anything, to install locally with brew, but when giving the PR branch name to the tag, it can't find and fails to proceed with installation. I'll remove this change to another place